openml/setups/functions.py

```diff
@@ -4,20 +4,23 @@
 import xmltodict
 
 from .setup import OpenMLSetup, OpenMLParameter
+from openml.flows import sklearn_to_flow, flow_exists
 
 
-def setup_exists(downloaded_flow, sklearn_model):
+def setup_exists(flow, model=None):
     '''
-    Checks whether a flow / hyperparameter configuration already exists on the server
+    Checks whether a hyperparameter configuration already exists on the server.
```

Comment:
please document why model can be none (i.e., flow.model is set)
please raise exception if flow.model is set and model is set, and these do not agree? (should never happen but still..)